packages/firestore/src/local/memory_target_cache.ts

```diff
@@ -191,14 +191,7 @@ export class MemoryTargetCache implements TargetCache {
     targetId: TargetId
   ): PersistencePromise<void> {
     this.references.addReferences(keys, targetId);
-    const referenceDelegate = this.persistence.referenceDelegate;
-    const promises: Array<PersistencePromise<void>> = [];
-    if (referenceDelegate) {
-      keys.forEach(key => {
-        promises.push(referenceDelegate.addReference(txn, key));
```

Comment:
We could keep this call... but all this does now is mark a document as non-orphaned if it was previously orphaned.  The keys now also added to the target as part of `notifyLocalViewChanges `.